### ISSUE TITLE
Honor `yaml.customTags` setting

### ISSUE THREAD

**Issue by psaxton** [42Crunch/vscode-openapi#41]
When this extension is enabled, other files such as cloud formation template.yaml files report errors for tags defined in the yaml.customTags setting. This does not happen with other extensions using the js-yaml engine such as redhat.vscode-yaml, ms-azuretools.vscode-docker, and ms-kubernetes-tools.vscode-kubernetes-tools.

![image](https://user-images.githubusercontent.com/15615330/74366345-179b4280-4d8d-11ea-88cd-2c6b9db95d75.png)

**Comment by ak1394** [comment]
Could you share a sample file please?

**Comment by psaxton** [comment]
See minimal files inline below. Note the files being affected are any yaml files, not just OpenAPI definitions. The settings.json file is the user profile settings with no workspace level settings. Disabling this extension clears up the errors in VS Code.

template.yaml
```
AWSTemplateFormatVersion: 2010-09-09
Transform: 'AWS::Serverless-2016-10-31'
Description: Customer API

Parameters:
  Environment:
    Type: String
    Description: The environment being deployed (e.g. 'test')
  Swagger:
    Type: String
    Description: The S3 location of the OpenAPI Spec for the API Gateway
  SwaggerVersion:
    Type: String
    Description: The Current Version of the Swagger Doc
  Stage:
    Type: String
    Description: The name of the stage to create
    Default: "test"

Resources:
  ClientApi:
    Type: AWS::Serverless::Api
    Properties:
      Name: !Sub '${Environment}-client-api - ${SwaggerVersion}'
      StageName: !Ref Stage
      DefinitionBody:
        Fn::Transform:
          Name: AWS::Include
          Parameters:
            Location: !Ref Swagger
```

settings.json:
```
{
    "yaml.customTags": [
        "!FindInMap sequence",
        "!GetAtt",
        "!GetAZs",
        "!ImportValue",
        "!Join sequence",
        "!Ref",
        "!Select sequence",
        "!Split sequence",
        "!Sub"
    ]
}
```

**Comment by ak1394** [comment]
Thanks, I see where the issue is!

**Comment by ak1394** [comment]
Fixed in 2.1.1
